### PR TITLE
Build and publish release via GitHub releases

### PR DESCRIPTION
**Pull request by buberdds**
[Sample](https://github.com/buberdds/explorer/releases/tag/v0.0.29 ) output triggered by tag push (fork)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          node-version: '18.x'
+          cache: yarn
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Build app
+        run: yarn build
+      - name: Create zip file
+        run: |
+          cd build/
+          zip -r ../explorer-build.zip .
```
Comment (tjanez) [inline .github/workflows/release.yml:36]: Let's name the release artifacts `oasis-explorer-<VERSION>.zip`.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        run: |
+          echo "VERSION=$(echo ${{ github.ref_name }} | sed 's/^v//')" >> "$GITHUB_OUTPUT"
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          files: |
+            explorer-build.zip
+          name: Oasis Explorer ${{ steps.vars.outputs.VERSION }}
+          body: |
+            For a list of changes in this release, see the [Change Log].
```
Comment (tjanez) [inline .github/workflows/release.yml:50]: Let's try to copy the part of the Change Log pertaining to the release here directly so it is more convenient for the users.

**Review by tjanez — COMMENTED** [review]
Great work!

File: .github/workflows/release.yml (hunk 2 above)
Comment (buberdds) [reply .github/workflows/release.yml:52]: The copy task is not enough. md file created by towncirer used in release notes looks like https://github.com/buberdds/explorer/releases/tag/v0.0.32 

1) Easy solution would be extract version related changes, but also adjust JINJA2 template to
```
- {{ lines[0] }} {{ referenced_issues }}
``` 
to keep entries in one line, but we will hit lint errors in CHANGELOG.md as lines will be too long

2) other option is to call helper script in node which extract version related changes from md file, but also tweak formatting (not sure we will handle all edge cases for formatting tweaks).  
sample output: https://github.com/buberdds/explorer/releases/tag/v1.1.0
sample script: https://github.com/buberdds/explorer/blob/master/internals/scripts/version-changelog.js

3) Keep current generic template

4) Some other JINJA2 workarounds maybe?

Comment (tjanez) [reply .github/workflows/release.yml:52]: I like option 2. And the script you wrote for the extraction looks pretty robust.

Comment (buberdds) [reply .github/workflows/release.yml]: PR updated, sample output https://github.com/buberdds/explorer/releases/tag/v1.2.0

Comment (tjanez) [reply .github/workflows/release.yml]: Great, thanks!

**Pull request merged**